kolibri/core/assets/src/views/KTooltip.vue

```diff
@@ -0,0 +1,97 @@
+<template>
+
+  <popper
+    :disabled="disabled"
+    :visibleArrow="false"
+    :options="options"
+    trigger="hover"
+    class="container d-ib"
+  >
+    <div
+      slot="reference"
+      class="d-ib"
+    >
+      <slot name="trigger"></slot>
```

Comment:
`trigger` makes me think of an event. maybe `target`? (or just keep `reference` like in popper?)